05-geometry-operations.Rmd

```diff
@@ -477,15 +477,14 @@ t2 = cast_all(polyg)
 ```
 
 Geometry casting of simple features geometry column (`sfc`) and simple features objects works the same as for single geometries in most of the cases. 
-One important difference is conversion between multi to non-multi types.
-As a result of this process, multi-objects are split into many non-multi objects.
+One important difference is the conversion between multi-types to non-multi-types.
+As a result of this process, multi-objects are split into many non-multi-objects.
 
 Table \@ref(tab:sfs-st-cast) shows possible geometry type transformations on simple feature objects.
-(Note the abbreviated geometry type names: POI, MPOI, LIN and GC for example refer to POINT, MULTIPOINT, LINESTRING and GEOMETRYCOLLECTION respectively.)
 Each input simple feature object with only one element (first column) is transformed directly into another geometry type.
 Several of the transformations are not possible, for example, you cannot convert a single point into a multilinestring or a polygon (so the cells `[1, 4:5]` in the table are NA).
-On the other hand, some of the transformations are splitting the single element input object into multi-element one.
-You can see that, for example, when you cast a multipoint consisting of five pairs of coordinated into a point.
+On the other hand, some of the transformations are splitting the single element input object into a multi-element object.
+You can see that, for example, when you cast a multipoint consisting of five pairs of coordinates into a point.
```

Comment:
@Nowosad To which row in the table are your referring to? 2nd row? But here the POI version only consists of four points, or am I just not understanding the table?